src/vs/code/browser/workbench/workbench.html

```diff
@@ -40,6 +40,20 @@
 		Object.keys(self.webPackagePaths).map(function (key, index) {
 			self.webPackagePaths[key] = `${baseUrl}/node_modules/${key}/${self.webPackagePaths[key]}`;
 		});
+
+		// Set up nls if the user is not using the default language (English)
+		const nlsConfig = {};
+		const storedLocale = window.localStorage.getItem('vscode.nls.configuration.language');
+		const locale = storedLocale || navigator.language;
+		if (!locale.startsWith('en')) {
```

Comment:
All other code here either gets bulled from the loader or is in the workbench bundle. Is there a specific reason for this code to be inlined and not another script that the loader pulls in / have it part of the main bundle?